lib/srv/db/snowflake/test.go

```diff
@@ -18,6 +18,7 @@
 
 package snowflake
 
+//nolint:goimports
```

Comment:
Why do we need the nolint here?

Reply:
We do not anymore. After I created this PR, the "inline comment" was moved from here on master. I'll remove the `nolint`.